main.go

```diff
@@ -377,7 +391,8 @@ func doRun(filePath string) error {
 // navLeft navigates to the next item to the left
 func navLeft(ctx context.Context) error {
 	chromedp.KeyEvent(kb.ArrowLeft).Do(ctx)
-	chromedp.WaitReady("body", chromedp.ByQuery)
+	// Could wait for the location to change instead of this Sleep.
+	time.Sleep(200 * time.Millisecond)
```

Comment:
yeah, I don't think we want arbitrary duration sleeps like that (as it's flaky, their effectiveness varying depending on the conditions), except when in a loop, when retrying things.

Reply:
I agree, I removed the `WaitReady("body",..)` because it had no effect, 

But I found that I needed a small delay, or my main loop would exit too soon, too often.

I did not want to tackle the *arbitrary* `Sleep(s)` in this PR, but I would like to propose an event based aproach. I have implemented this in a separate re-implmentation (node.js+puppeteer) which works really well.

 - **For navigation:** we can use `chromedp.ListenTarget` and get a callback when `page.EventNavigatedWithinDocument` or `target.EventTargetInfoChanged` occur, this is more reliable, and is much *tighter* from a timing perspective.
- **For download initiation:** Similarly, there is a `page.EventDownloadWillBegin` event which can obviate the directory watching loop in `(*Session) download()`

If you agree I can make a separate pull request to demonstrate this idea.

Reply:
Sure, I'm very interested to see your approaches in another PR. But in the meantime, let's leave navLeft as it was. The fact that navLeft sometimes does not work for you is unrelated to the change that removes the screenshots, right?

Reply:
So here's my empirical testing on this point:

1) If I remove your time.Sleep, I have the problem that it only gets one photo, and then it stops.
2) If re-add your time.Sleep, it indeed seems to help (but we don't want that).
3) If I re-add instead the previous code (i.e. chromedp.WaitReady("body", chromedp.ByQuery)), it also seems to work. So if what you say is true (that chromedp.WaitReady("body", chromedp.ByQuery) does nothing), maybe it still has the fortunate side-effect of slowing things down (and making navLeft work) just because we're spending some CPU cycles  doing "something"?

Reply:
Ok, I agree, that arbitray Sleep is *bad*, so until we have a *better condition* to wait for, I will restore navLeft as it was.